src/NuGet.Core/NuGet.Versioning/VersionRangeComparer.cs

```diff
@@ -56,7 +56,9 @@ public VersionRangeComparer(IVersionComparer versionComparer)
 
         internal static IVersionRangeComparer VersionReleaseMetadata { get; } = new VersionRangeComparer(VersionComparer.VersionReleaseMetadata);
 
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
         public static IVersionRangeComparer Get(VersionComparison versionComparison)
+#pragma warning restore CS1591 // Missing XML comment for publicly visible type or member
```

Comment:
Same as the other Nuget.Versioning file, this is easy to document, which I'd rather see than kicking the can down the line.